prime-router/metadata/tables/local/sender_automation_value_set_row.csv

```diff
@@ -0,0 +1,1628 @@
+id,sender_automation_value_set_id,code,display,version
+1,1,F,Female,
```

Comment:
There is some redundancy here in the way all lookup tables are stored in the database.  You have a similar setup here with a table that list the tables and one that has the rows.  This is the same pattern implemented in the lookup tables, so you could simplify this data a bit.  My suggestion is to create a lookup table for each value set.  E.g. one value set's CSV file would be
```
F, Female
F,Woman
F,F
M,Male
M,Man
M,M
U,U
U,UNK
U,UNKNOWN
```
Note order is retained, hence you do not need the ID column, and the value set is identified by its lookup table name.
With this setup, you can manage that one lookup table with the current set of CLI tools too.  This also removes the need for the `sender_automation_value_set` table.

Reply:
@carlosfelix2 we were very "on the fence" about whether or not to go with one or many tables. We went with one table for this MVP pass because we realized that if we had _many_, we would have chain 12+ calls from the UI to the API layer. Extending the API layer to "aggregate" that chaining (or extend it at all) was something we were hoping to avoid doing (aside from minor tweaks we found along the way)

Reply:
also, the ID columns are _solely_ to relate `sender_automation_value_set` (which is just header metadata that the UI uses) to `sender_automation_value_set_row` (the actual data). Even if we were going to many tables, we would still need to preserve that relationship